server.go

```diff
@@ -689,6 +689,8 @@ func newServer(listenAddrs []net.Addr, chanDB *channeldb.DB, cc *chainControl,
 		RotateTicker:         ticker.New(discovery.DefaultSyncerRotationInterval),
 		HistoricalSyncTicker: ticker.New(cfg.HistoricalSyncInterval),
 		NumActiveSyncers:     cfg.NumGraphSyncPeers,
+		MinimumBatchSize:     10,
```

Comment:
This commit should be squashed with the commit that introduced the variables (994df1bb99c41319b6c5f6963b5e91a8e1092cf1) so that each individual commit compiles.